static/js/main.js

```diff
@@ -64,14 +65,21 @@ ethereumCp.on('success', function (e) {
 });
 
 $('h1.ui.header').transition('pulse');
+if (theme && theme === 'dark')  {
+    $('.light').toggleClass('dark').removeClass('light');
+    $('#github-logo').attr('src', 'static/images/gh-mark-light.svg');
+    $('div#dark-mode').addClass('checked');
```

Comment:
According to Semantic UI docs, I think this line must be replaced with this:
```
$('#dark-mode.checkbox').checkbox('attach events', '.check.button', 'check');
```

Reply:
Well, I tested out this, and this too was not working. But the below line is working nicely (and I got this idea from the Semantic UI docs and Developer Tools)
```javascript
$('div#dark-mode input').attr('checked', 'checked');
```
I am updating the corresponding .js and .min.js files and will update this pull request soon.